CHANGELOG.md

```diff
@@ -9,244 +9,213 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
-
+- Added new `UnrepeatedDistributedSampler` and `IndexBatchSampler6139Wrapper` for tracking distributed predictions ([#7215](https://github.com/PyTorchLightning/pytorch-lightning/pull/7215))
 
 - Added `trainer.predict(return_predictions=None|False|True)` ([#7215](https://github.com/PyTorchLightning/pytorch-lightning/pull/7215))
 
-
 - Added `BasePredictionWriter` callback to implement prediction saving ([#7127](https://github.com/PyTorchLightning/pytorch-lightning/pull/7127))
 
-
 - Added `trainer.tune(scale_batch_size_kwargs, lr_find_kwargs)` arguments to configure the tuning algorithms ([#7258](https://github.com/PyTorchLightning/pytorch-lightning/pull/7258))
 
-
 - Added `tpu_distributed` check for TPU Spawn barrier ([#7241](https://github.com/PyTorchLightning/pytorch-lightning/pull/7241))
 
-
 - Added device updates to TPU Spawn for Pod training ([#7243](https://github.com/PyTorchLightning/pytorch-lightning/pull/7243))
 
-
 - Added warning when missing `Callback` and using `resume_from_checkpoint` ([#7254](https://github.com/PyTorchLightning/pytorch-lightning/pull/7254))
 
+- DeepSpeed single file saving ([#6900](https://github.com/PyTorchLightning/pytorch-lightning/issues/6900))
 
-- Improved verbose logging for `EarlyStopping` callback ([#6811](https://github.com/PyTorchLightning/pytorch-lightning/pull/6811))
-
-
-- Fix yaml loading with PyYAML=5.4.x ([#6666](https://github.com/PyTorchLightning/pytorch-lightning/issues/6666))
+- Added Training type Plugins Registry (
+    [#6982](https://github.com/PyTorchLightning/pytorch-lightning/issues/6982),
+    [#7063](https://github.com/PyTorchLightning/pytorch-lightning/issues/7063),
+    [#7214](https://github.com/PyTorchLightning/pytorch-lightning/issues/7214),
+    [#7224](https://github.com/PyTorchLightning/pytorch-lightning/issues/7224)
+)
 
+- Add `ignore` param to `save_hyperparameters` ([#6056](https://github.com/PyTorchLightning/pytorch-lightning/issues/6056))
 
 ### Changed
 
-
 - Changed `LightningModule.truncated_bptt_steps` to be property ([#7323](https://github.com/PyTorchLightning/pytorch-lightning/pull/7323))
 
-
 - Changed `EarlyStopping` callback from by default running `EarlyStopping.on_validation_end` if only training is run. Set `check_on_train_epoch_end` to run the callback at the end of the train epoch instead of at the end of the validation epoch ([#7069](https://github.com/PyTorchLightning/pytorch-lightning/pull/7069))
 
-
 - Renamed `pytorch_lightning.callbacks.swa` to `pytorch_lightning.callbacks.stochastic_weight_avg` ([#6259](https://github.com/PyTorchLightning/pytorch-lightning/pull/6259))
 
+- Refactor `RunningStage` and `TrainerState` usage (
+    [#4945](https://github.com/PyTorchLightning/pytorch-lightning/pull/4945),
+    [#7173](https://github.com/PyTorchLightning/pytorch-lightning/pull/7173))
+    * Added `RunningStage.SANITY_CHECKING`
```

Comment:
Changed section shouldnt have these added lines

Reply:
seems not to need as MarkDown and RST used in doc has different rules